python/rmm/_lib/memory_resource.pyx

```diff
@@ -651,3 +671,18 @@ def disable_logging():
     for i, each_mr in _per_device_mrs.items():
         if isinstance(each_mr, LoggingResourceAdaptor):
             set_per_device_resource(i, each_mr.get_upstream())
+
+
+def get_log_filenames():
+    """
+    Returns the log filename (or `None` if not writing logs)
+    for each device in use.
+    """
+    global _per_device_mrs
+
+    return [
```

Comment:
Thank you, this is a very helpful addition.  One question though, just to clarify my understanding:
```
Use `rmm.get_log_filenames()` to get the log file names corresponding to each device.
```
so if I wanted to find the logfile that corresponds to a particular device a user specified using `CUDA_VISIBLE_DEVICES`, I'd have to cross reference the ordering in `CUDA_VISIBLE_DEVICES` and map the first device ID in their list to the file with `.dev0`, the second ID to the file with `.dev1`, etc.?

Ex. `CUDA_VISIBLE_DEVICES=2,1,0` means user dev 2 = file with `.dev0`, user dev 1 = `.dev1`, etc.